includes/class-indieauth-authorization-endpoint.php

```diff
@@ -241,11 +241,11 @@ public function verify( $request ) {
 			return new WP_OAuth_Response( 'invalid_grant', __( 'Invalid authorization code', 'indieauth' ), 400 );
 		}
 		$user = get_user_by( 'id', $token['user'] );
-		if ( $token['expiration'] <= time() ) {
+		if ( $token['exp'] <= time() ) {
```

Comment:
:question: is this the response from the token endpoint?

Reply:
No. This is the authorization endpoint. I use the same expiration code for authorization codes as I do for tokens, so I standardized using exp.

Reply:
I'm not returning it.